skylarktest/skylarktest.go

```diff
@@ -134,9 +134,10 @@ func freeze(thread *skylark.Thread, _ *skylark.Builtin, args skylark.Tuple, kwar
 }
 
 // DataFile returns the effective filename of the specified
-// test data resource.  The function abstracts differences between
-// 'go build', under which a test runs in its package directory,
-// and Blaze, under which a test runs in the root of the tree.
-var DataFile = func(pkgdir, filename string) string {
+// test data resource. The symbol allows for overriding of the
```

Comment:
Let's keep the previous comment and just change Blaze to "Bazel and Blaze".
The additional sentence is fine, but can be shortened to:

"The default behavior works for go build."

Reply:
> Let's keep the previous comment

Maybe it is me not being a native speaker but IMO the previous comment is plain wrong. The function does not abstract differences [...] but abstracts one specific way of building the file path (for `go build`). The assignable symbol then enables changing that behavior.

Reply:
You're right that each anonymous function abstracts only one specific way, but think of it from the point of view of the client: DataFile, although it is technically a variable containing one of three functions, acts like a single function that hides (abstracts) each build system's peculiar way of locating data.

Reply:
Fair enough. Still I try to make it a bit more descriptive.